Tests/ScipioKitTests/RunnerTests.swift

```diff
@@ -297,7 +300,7 @@ final class RunnerTests: XCTestCase {
         XCTAssertTrue(fileManager.fileExists(atPath: storageDir.appendingPathComponent("ScipioTesting").path))
 
         addTeardownBlock {
-            try self.fileManager.removeItem(at: storageDir)
+            try FileManager.default.removeItem(at: storageDir)
```

Comment:
```swift
addTeardownBlock { [fileManager] in 
    try fileManager.removeItem(at: storageDir)
}
```

Reply:
I tried this, but `FileManager` is a non-sendable class, so we can't capture them for the `@Sendable` closures.

Reply:
Hmm...
The current code is inconsistent, because there are similar code, `FileManager.default` and `fileManager`, in the test code. 
And more, the test code depends on the assumption that `fileManger` is `FileManager.default`, and it should be avoided.

There are three options.
1. Replace all `fileManager` with `FileManager.default`.
2. Stop using `addTeardownBlock` and use just `defer` for the clean up.
3. Add `unchecked Sendable` to the closure. This is test codes, so I expect there are no major issues related to data races. (However, we should not use it as much as possible). 

What do you think?

Reply:
In final, I removed `defer` / `addTeardownBlock` for these cases.
It's enough to call on the bottom of methods.